src/themes/elife/styles.css

```diff
@@ -215,7 +215,7 @@
     }
   }
 
-  & .download-link-wrapper {
+  & .pre-header-wrapper {
```

Comment:
I think we need to keep it similar to the journal site so if we call it something like ```.content-header```